backend/app/api/v1/commons/telco.py

```diff
@@ -0,0 +1,56 @@
+from datetime import date
+import pandas as pd
+from app.services.splunk import SplunkService
+import app.api.v1.commons.hasher as hasher
+from datetime import datetime, timezone
+
+
+async def getData(start_datetime: date, end_datetime: date, configpath: str):
+    test_types = ["oslat", "cyclictest", "cpu_util", "deployment", "ptp", "reboot", "rfc-2544"]
+    jenkins_url = "https://ci-jenkins-csb-kniqe.apps.ocp-c1.prod.psi.redhat.com/job/ocp-far-edge-vran-tests/"
```

Comment:
Do we want to have this URL in the public?
I would rather have it as a config

Reply:
Yes it can be. its just a placeholder for now with limited access.

Reply:
Just as an FYI @vishnuchalla URLs/Hostnames are sensitive and I would prefer we not have that in our public github.

Reply:
Moved to configuration and updated the PR. Please take a look.